### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by KrasiStoyanov**
Replaced our custom css class ".button" with Bootstrap's ".btn". Made all the active buttons in the landing page with this class. Extended Bootstrap's possibilities by adding custom .btn-[color] classes, which are based on the button's color. Also created classes like ".invert-color" to invert its colors on hover. Also removed all the links for our job offers and have commented the files for these offers.

### CODE REVIEW

File: src/sass/bootstrap-mods/buttons.scss [hunk 1]
```diff
@@ -1,5 +1,84 @@
...
+@import "core/settings";
 
+// BUTTON STYLES
+.btn {
+    border: 0;
+    @include transition-full(background .2s ease-in-out);
+
+    &:active,
+    &:active:focus,
+    &:focus {
```
Comment (ivank) [inline src/sass/bootstrap-mods/buttons.scss]: there are some more styles here that we can add here for this:
```css
&:focus, &.focus, &:active:focus, &:active.focus, &.active:focus, &.active.focus {
...
}
```

File: src/templates/pages/job/graphic-designer.hbs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{#extend "layout"}}
+<!-- {{#extend "layout"}}
```
Comment (ivank) [inline src/templates/pages/job/graphic-designer.hbs]: Can just delete this part. We can use git to get old code if we need to :)

File: src/templates/pages/jobs.hbs [hunk 3]
```diff
@@ -38,7 +38,7 @@
                             </div>
                         </div>
                     </div>
-                    <a href="/join-the-force.html">
+                    <!-- <a href="/join-the-force.html">
```
Comment (ginovski) [inline src/templates/pages/jobs.hbs]: We don't need to comment code, we can just delete it. Git keeps track of all files, so we can revert if we need :)

**Comment by ginovski** [comment]
LGTM

**Pull request merged**